### PR TITLE
settings: improve dialog selet lang/time format.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About:
Remove space contourn popup select, get better result.

### CODE REVIEW

**Comment by marcoluc97** [comment]
Hi @Zangetsu38 I do not know if this is useful but if you are excluding NPXS10015 because is the Settings app maybe is better to add also NPXS10016, or i'm just wrong so ignore this.

[PSVita Dev Wiki Title ID](https://playstationdev.wiki/psvitadevwiki/index.php?title=Title_ID)

**Comment by Zangetsu38** [comment]
> 
> 
> Hi @Zangetsu38 I do not know if this is useful but if you are excluding NPXS10015 because is the Settings app maybe is better to add also NPXS10016, or i'm just wrong so ignore this.
> 
> [PSVita Dev Wiki Title ID](https://playstationdev.wiki/psvitadevwiki/index.php?title=Title_ID)

yes, i using only one setting, no other

File: vita3k/gui/src/app_selector.cpp [hunk 1]
```diff
@@ -31,7 +31,13 @@ using namespace std::string_literals;
 namespace gui {
 
 void init_user_apps(GuiState &gui, HostState &host) {
-    gui.apps_background.clear();
+    if (gui.live_area.settings) {
+        for (const auto &app : gui.apps_list_opened) {
+            if (app != "NPXS10015")
+                gui.apps_background.erase(app);
+        }
+    } else
```
Comment (IllusionMan1212) [inline vita3k/gui/src/app_selector.cpp:39]: please add curly braces here, it makes it more readable especially since the if statement has braces

**Review by IllusionMan1212 — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/app_selector.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/app_selector.cpp:39]: Is not needed with one return value

Comment (IllusionMan1212) [reply vita3k/gui/src/app_selector.cpp:39]: it works yes, but its a lot more readable to have braces

Comment (Zangetsu38) [reply vita3k/gui/src/app_selector.cpp:39]: no really more readable :) i no want touch it is fine here

Comment (Zangetsu38) [reply vita3k/gui/src/app_selector.cpp:39]: request approve, i no want change it here, is good like that

Comment (Zangetsu38) [reply vita3k/gui/src/app_selector.cpp]: review again, this code change is totaly removed, no need anymore ^^

**Review by IllusionMan1212 — APPROVED** [review]

**Pull request merged**